src/main/java/com/google/enterprise/cloudsearch/fs/FsRepository.java

```diff
@@ -1184,7 +1244,7 @@ private void getFileContent(Path doc, FileTime lastAccessTime, Item item,
       RepositoryDoc.Builder operationBuilder) throws IOException {
     String mimeType = getDocMimeType(doc);
     item.getMetadata().setMimeType(mimeType);
-    item.setItemType(ItemType.CONTENT_ITEM.name());
+    item.setItemType(getItemType(doc).name());
```

Comment:
Why did this need to change?

Reply:
So that getItemType is the one place where we decide on types.